sql/moz-fx-data-shared-prod/jira_service_desk_syndicate/dataset_metadata.yaml

```diff
@@ -22,7 +22,13 @@ syndication:
     - request
```

Comment:
syndicates are already write-restricted so no need to update the dataset_base_acl here